modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -241,6 +300,8 @@ struct ROIList: public ::testing::Test {
             auto plugin        = cv::gimpl::ie::wrap::getPlugin(params);
             auto net           = cv::gimpl::ie::wrap::readNetwork(params);
             setNetParameters(net);
+            dim_for_reshape = 70;
+            setReshape(net, dim_for_reshape);
```

Comment:
I don't think it's good to have additional feature tested in the existed test. Would be better to make a new one.
Try to make a new test fixture with things inherited from ROIList